tests/Microsoft.Identity.Test.Integration.Win8/DeviceAuthenticationTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Json.Linq;
+using Microsoft.Identity.Test.LabInfrastructure;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Identity.Test.Integration.net45.HeadlessTests
+{
+    [TestClass]
+    public class DeviceAuthenticationTests
+    {
+        private const string _claims = "{\"access_token\":{\"deviceid\":{\"essential\":true}}}";
+        private const string _publicClientId = "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0";
+        private const string _deviceAuthuser = "idlabca@msidlab8.onmicrosoft.com";
+
+        [TestMethod]
+
+        public void PKeyAuthNonInteractiveTest()
```

Comment:
What happens if we run this test on a Win10 developer machine? We discussed offline that tests should not fail on developer machines, but it is acceptable to finish as inconclusive via `Assert.Inconclusive`. This is what we do on the cache ext project and works quite well.

Reply:
I missed this. will add